sjcl/build.boot

```diff
@@ -0,0 +1,40 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
+
+(require '[boot.task-helpers]
```

Comment:
I don't think requiring `boot.task-helpers` here is needed. It's not explicitly used anywhere and Boot loads that namespace anyway.